recipes/mmquant/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "mmquant" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://bitbucket.org/mzytnicki/multi-mapping-counter/get/{{ version }}.tar.gz
+  sha256: d2d6c5440345ad66095e27fcdb2fe5bdf8e15af1d3005953da78630b3680f6b7
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - make
```

Comment:
Does this need `make` to run or is this here by mistake?

Reply:
@dpryan79 Sorry, newbie here.
The tool is available as a series of C++ files, so yes, you need to "make" it to run.